articles/hosted-pages/login/index.md

```diff
@@ -14,20 +14,29 @@ Auth0's Hosted Login Page is the most secure way to easily authenticate users fo
 
 ### How Does the Hosted Login Page Work
 
-Auth0 shows the Hosted Login Page whenever something (or someone) triggers an authentication request, such as calling the `/authorize` endpoint (OIDC/OAuth) or sending a SAML login request. Users will see the Hosted Login Page, typically with either the Lock widget or with your custom UI. Once they login, they will be redirected back to your application.
+Auth0 shows the Hosted Login Page whenever something (or someone) triggers an authentication request, such as calling the `/authorize` endpoint (OIDC/OAuth) or sending a SAML login request. It can also be accessed via a request, in the following format:
+
+```html
+https://${account.namespace}.auth0.com/login?client=${account.clientId}
+```
+
+Users will see the Hosted Login Page, typically with either the Lock widget or with your custom UI. Once they login, they will be redirected back to your application.
 
 ::: warning
 If the incoming authentication request includes a `connection` parameter that uses an external identity provider (such as a social provider), the Hosted Login Page will not display. Instead, Auth0 will direct the user to the [identity provider's](/identityproviders) login page.
 :::
 
-#### SSO
+#### Single Sign-On (SSO)
 
 When SSO is desired, a client should use the Hosted Login Page for logins rather than an embedded login solution. When a user logs in via the Hosted Login Page, a cookie will be created and stored. On future calls to the `authorize` endpoint, the cookie will be checked, and if SSO is achieved, the user will not ever be redirected to the Hosted Login Page. They will see the page only when they need to actually login. 
 
-This behavior occurs without modification to the actual Hosted Login Page. The process involves enabling SSO for the client in the [Dashboard](${manage_url}) and then using the `authorize` endpoint with `?prompt=none` for silent SSO.
+This behavior occurs without modification to the actual Hosted Login Page. The is a simple two step process:
+
+1. Enable SSO for the client in the [Dashboard](${manage_url}) (Go to the Client's Settings, then scroll down to the "Use Auth0 instead of the IdP to do Single Sign On" setting and toggle it on.
+1. Use the `authorize` endpoint with `?prompt=none` for silent SSO.
```

Comment:
Link to the doc maybe? It's the silent authentication one https://auth0.com/docs/api-auth/tutorials/silent-authentication. Also, it's better if we link to the endpoints instead of just mentioning them.